README.md

```diff
@@ -71,7 +73,18 @@ models:
 
 *Read more about using custom schemas in dbt [here](https://docs.getdbt.com/docs/building-a-dbt-project/building-models/using-custom-schemas).*
 
-### Contributions ###
+### Running on Live vs Test Customers
+By default, this package will run on non-test data (`where livemode = true`) from the source Stripe tables. However, you may want to include and focus on test data when testing out the package or developing your analyses. To run on _only_ test data, add the following configuration to your `dbt_project.yml` file:
```

Comment:
Did you intend for `_only_` to have the underscores?

Reply:
i did at the time but maybe i should just remove...

Reply:
Yeah I like it italicized, but let's remove the underscores.